groups/alteryx-sandbox/nlb.tf

```diff
@@ -0,0 +1,99 @@
+# Load Balancer
+resource "aws_lb" "alteryx_sandbox" {
+  name               = "alb-${var.application}-${var.application_environment}"
+  internal           = true
+  load_balancer_type = "network"
+  security_groups    = [aws_security_group.ec2_alb.id]
+  subnets            = local.alteryx_subnet_id
+
+  tags = {
+    Environment = var.environment
+    Service     = var.application
+  }
+}
+
+# Target configuration for HTTP / port 80
+resource "aws_lb_target_group" "alteryx_sandbox_web" {
+  name        = "${var.application}-${var.application_environment}-web"
+  port        = 80
+  protocol    = "HTTP"
+  target_type = "ip"
+  vpc_id      = data.aws_vpc.vpc.id
+
+  health_check {
+    healthy_threshold   = 2
+    unhealthy_threshold = 2
+    timeout             = "5"
+    path                = "/gallery/api/status/ping/"
+    protocol            = "HTTP"
+    interval            = 30
+    matcher             = "200"
+  }
+}
+
+resource "aws_lb_target_group_attachment" "alteryx_sandbox_web" {
```

Comment:
Resource names need to be unique - see https://github.com/companieshouse/alteryx-terraform/pull/34/files#r1582738476